autoconnect/autoconnect-settings/src/app_state.rs

```diff
@@ -51,7 +51,9 @@ impl AppState {
         let fernets: Vec<Fernet> = crypto_key
             .split(',')
             .map(|s| s.trim().to_string())
-            .map(|key| Fernet::new(&key)._or_else(|| panic!("Invalid {ENV_PREFIX}_CRYPTO_KEY")))
+            .map(|key| {
+                Fernet::new(&key).unwrap_or_else(|| panic!("Invalid {ENV_PREFIX}_CRYPTO_KEY"))
```

Comment:
Huh, not sure why that didn't get flagged by the analyzer when I loaded you branch.

_Goes and tries to figure out what's broken_